Overview.src.html

```diff
@@ -688,25 +705,25 @@ <h4 id=terminology-headers>Headers</h4>
   <tr>
    <td>"<code title>fetch</code>"
    <td>""
-   <td>"<code title>subresource</code>"
+   <td>"<code title>fetch</code>"
    <td><code title>connect-src</code>
-   <td><code title=dom-global-fetch>fetch()</code>, Cache API?
+   <td><code title=dom-global-fetch>fetch()</code>, <code>XMLHttpRequest</code>, Cache API?
```

Comment:
The main difference is that it would no longer be omitted by default for `fetch()`. And there'd be no way to omit it (we'd have to add something new for that). I think I'd be okay with that for this header, personally. Especially since it aligns a bunch of things.